eng/pipelines/templates/jobs/archetype-sdk-client.yml

```
@@ -162,9 +162,15 @@ jobs:
           ward scan -d $(Build.SourcesDirectory) -c $(Build.SourcesDirectory)/eng/.docsettings.yml
         displayName: 'Verify Readmes'
 
+      - pwsh: |
+            $diffs = git diff origin/master HEAD --name-only -- *.md
+            echo "##vso[task.setvariable variable=diffMarkdown]$diffs"
+        displayName: 'Find diff files'
+
       - template: ../../../common/pipelines/templates/steps/verify-links.yml
         parameters:
           Directory: sdk/${{ parameters.ServiceDirectory }}
+          Urls: $(diffMarkdown)
```

Why not just put this inline like:
```suggestion
          Urls: (git diff origin/master HEAD --name-only -- *.md)
```